db.json

```diff
@@ -0,0 +1,58 @@
+[
+  {
+    "initiallyLost": true,
+    "itemId": "c871d719-6a20-43da-9f38-98069232f6a0",
+    "itemTitle": "Watch",
+    "itemDescription": "Red watch with a black lether strap.",
```

Comment:
I recommend not using the item prefix in keys in your data structure. This is redundant. You will then use it like this `item.itemDescription` instead of `item.description`

Reply:
Thank you for review!
I changed `itemTitle` to `title` and `itemDescription ` to `description `.  But I need `itemId `, because I have also `userId`.

Reply:
You can use id, since it is different from userId. You also have to commit and push after you implement the changes, so they can be seen in this pull request.